### PR TITLE
Changed confusing labels #625

### PR DESCRIPTION
**Pull request by jinamshah**
- Changed “List View” to “Crop List”
- Changed “Calendar View” to “Crop Calendar”
- Changed “View My List” to “My Selected Crops”

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.217.220.9:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.217.220.9:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.217.220.9:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.217.220.9:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/shared/ProgressButtonsInner.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ const ProgressButtonsInner = ({
             onClick={() => (progressRedux === 4 ? setMyCoverCropActivationFlag() : changeProgress('increment'))}
             disabled={isDisabledNext || (progressRedux === 4 && selectedCropIdsRedux.length === 0)}
           >
-            {progressRedux === 4 ? 'VIEW MY LIST' : 'NEXT'}
+            {progressRedux === 4 ? 'MY SELECTED CROPS' : 'NEXT'}
```
Comment (Adamws33) [inline src/shared/ProgressButtonsInner.js:90]: changing this text has left the button with 2 rows of text.  Can you please adjust the size of the button so we can see how it looks when the text is on just one line.  I can see we are adjusting the size of the button for this text already i just think it needs to be a little bigger so the text is on one line.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.217.220.9:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://48.217.220.9:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by Adamws33 — APPROVED** [review]

**Pull request merged**